discussion_agents/cog/functional/react.py

```diff
@@ -31,32 +31,27 @@ def _build_agent_prompt(question: str, scratchpad: str, examples: str , prompt_t
     return prompt
 
 
-def _prompt_agent(llm: BaseChatModel, question: str, scratchpad: str, examples: str, prompt_template: str) -> str:
+def _prompt_agent(llm: BaseChatModel, question: str, scratchpad: str, examples: str, prompt_template: str, stop: Union[List[str], None] = None) -> str:
```

Comment:
I am not sure if we need to add stop into the param . I guess we can but the default should be '\n'

Reply:
i got you